src/vivarium_gates_nutrition_optimization_child/components/observers.py

```diff
@@ -308,3 +308,33 @@ def setup(self, builder: Builder) -> None:
         self.entity_type = self.disease_model.cause_type
         self.entity = self.disease_model.cause
         self.transition_stratification_name = f"transition_{self.disease}"
+
+    # We want diseaste state to be categories instead of the state_ids
```

Comment:
The two methods below look like they could (should?) be in the DiseaseObserver class. Is there a nuance as to why they aren't? This is a custom component so I'm not just not making the connection

Reply:
They _are_ in the disease class but I'm overwriting them